delivery-receipts-api/src/main/java/uk/gov/di/authentication/deliveryreceiptsapi/entity/NotifyDeliveryReceipt.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.di.authentication.deliveryreceiptsapi.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class NotifyDeliveryReceipt {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("reference")
+    private String reference;
+
+    @JsonProperty("to")
+    private String to;
+
+    @JsonProperty("status")
+    private String status;
+
+    @JsonProperty("created_at")
+    private String created_at;
```

Comment:
I think we should be using camel case rather than snake case, in java code at least.

Reply:
Yep agreed. I will update